### DISCUSSION TITLE
How to create an array type on Supabase UI?

### DISCUSSION THREAD

**Discussion by junaid33** [supabase/supabase#342] [Questions]
Hello,

I am trying to migrate our existing project to Supabase. When I click create column on a table, I don't see an array type. Any examples or guides explaining how to create array type columns?

**Comment by inian** [comment]
Hi @junaid33, you can add native PG arrays using the SQL Editor. It’s also supported by the client library. I assume you are looking for support in the table editor too?

**Reply by inian** [reply]
We have launched support for creating array types in the table editor now too!
